docs/llm.md

```
@@ -19,6 +19,42 @@ Other agents do not yet have LLM mapping and will not receive any LLM configurat
 ollama pull qwen3:14b
 ```
 
+!!! note "Linux: expose Ollama to Docker"
+    On Linux, Ollama binds to `127.0.0.1` by default. Docker containers reach the host via the Docker bridge gateway (for example `172.17.0.1` on the default Docker bridge), so the default binding will refuse connections.
+
+    **If running Ollama manually:**
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)